### DISCUSSION TITLE
Text under QR Code

### DISCUSSION THREAD

**Discussion by find-usdk** [photobooth-app/photobooth-app#366] [General]
Is there a way i can add a text under the QR code?

I would like to type this text under QR code: 
"Connect to WiFi Photobooth before scanning QR code"

**Example:**
![image](https://github.com/user-attachments/assets/fa95c2b8-e0dc-4f6c-aec6-21b181f491b2)

**Comment by mgineer85** [comment]
Currently not possible. Maybe you could hack it with css :after elements ?
Feels free to send a pull request so it can be added later. I think it would be a good addition

**Reply by mgineer85** [reply]
Added this option in the current development version. Once v5 is released, you can define two texts:

![image](https://github.com/user-attachments/assets/4ddb09d7-0ed5-46c9-af7e-00de1547ccd2)

**Reply by find-usdk** [reply]
Wow it´s great 👍  

Do we know approximately when v5 will be released?

**Reply by mgineer85** [reply]
It’s major revision under the hood and needs testing as well as the final implementation of the wigglecam feature.
No eta yet, but there is time in between the years to work on it ☺️
